my-app/src/App.js

```diff
@@ -46,16 +54,21 @@ class Board extends Component {
       let cells = [];
       for (let i in row) {
         cells.push(<td key={`${row[i].x}-${row[i].y}`}>
-                    <button x={row[i].x} y={row[i].y} className={row[i].value} onClick={(event) => this.handleClick(row[i])}>{row[i].x},{row[i].y}</button>
+                    <button x={row[i].x} y={row[i].y} className={row[i].value} onClick={(event) => this.handleClick(row[i])}>_</button>
```

Comment:
You should try to keep lines of code at less than 80 character to ensure readability (this code is cut off in the github UI, for example).  I also recommend avoiding aligned indents; simply using two space indents on successive lines is generally enough to communicate code organization (and then maintainers of the code are not forced to tediously, perfectly align code changes that may be necessary):
```js  
rows() {
  return this.props.rows( (row, index) => {
    return (
      <td key={`row-${index}`}>
        <button x={row.x} y={row.y} className={row.value} 
          onClick={(event) => this.handleClick( row )>_</button>
      </td>
    )
  })
}

return <tr key={index}>{this.rows()}</tr>
```
I changed the keys to ensure they will be unique in the DOM.  I refactored to split the responsibility of generating the rows out of this context (whose responsibility appears to be the rendering of the **row**). I also updated the variable name from `row` to `rows`, which makes the intention of the variable clearer (it holds many things, not just one), and allowed me to use the more meaningful `row` identifier as I map over the `rows`.  You'll note that I used `map` above - any time I see the pattern:
```js
// temporary variable declaration
// for/forEach
//  change/update temporary variable
```
I replace it with the more concise `map` (or other functional approach).